### DISCUSSION TITLE
Recognition of certain things on MacOS.

### DISCUSSION THREAD

**Discussion by Neosake** [CISOfy/lynis#1165] [Q&A] [answered]
I was using Lynis on my Mac, and was wondering if there is a solution under way to get Lynis on MacOS to recognize some things.

For example; ssh, databases (sqlite per say), automation tooling (Not recognizing Automator as the automation tool.) and not sure whether it pertains to you guys or not, but Homebrew's audit feature being built in (Maybe there is a way to get this wonderful tool to somehow acknowledge it through a shell script?).

**Comment by mboelen** [comment, marked as answer]
Yes, all the time we add new things and we welcome this feedback.

How to get something added? Report each item as an individual issue on GitHub. For example: "Automator is not detected on macOS" as its title. Then include in the description the details on how Automator could be detected, like the presence of a binary or an active process.

If you are familiar with shell scripting, then have a look at the code on the existing items that are already detected. If you are a bit handy, then typically you can create the new detection yourself and submit it into a pull request.

**Reply by Neosake** [reply]
Noted! I will be sure to include this in future questions, and thank you so much for the response. I will work on finding possible implementations and be sure to help out any way that I could.
